qcodes/tests/dataset/test_nested_measurements.py

```diff
@@ -167,6 +167,7 @@ def subscriber(results: List[Tuple[VALUE]], length: int,
     return subscriber
 
 
+@pytest.mark.flaky(reruns=5)
 @pytest.mark.serial
```

Comment:
we should alsao remove this marker in this PR, right?

Reply:
Yes if this actually works (as you can see from the outcome it probably does not)

Reply:
Since the serial jobs are still needed for azure pipelines on windows this will be left as is